src/types/optional_string.rs

```diff
@@ -0,0 +1,41 @@
+#[derive(Debug, Default, Clone)]
```

Comment:
Do we really need this file? I think, it's a bit weird

Reply:
Deleted